src/ensembl/src/content/app/browser/browser-region-editor/BrowserRegionEditor.tsx

```diff
@@ -316,15 +309,18 @@ export const BrowserRegionEditor = (props: BrowserRegionEditorProps) => {
   );
 };
 
-const mapStateToProps = (state: RootState) => ({
-  activeGenomeId: getBrowserActiveGenomeId(state),
-  chrLocation: getChrLocation(state) as ChrLocation,
-  genomeKaryotypes: getGenomeKaryotypes(state) as GenomeKaryotype[],
-  regionEditorActive: getRegionEditorActive(state),
-  regionFieldActive: getRegionFieldActive(state),
-  regionValidationInfo: getRegionValidationInfo(state),
-  regionValidationLoadingStatus: getRegionValidationLoadingStatus(state)
-});
+const mapStateToProps = (state: RootState) => {
+  return {
+    activeGenomeId: getBrowserActiveGenomeId(state),
+    chrLocation: getChrLocation(state) as ChrLocation,
+    genomeKaryotypes: getGenomeKaryotypes(state) as GenomeKaryotype[],
```

Comment:
The selector says that it can return null. Why do we convince it here that it can’t?
_(Same for `getChrLocation`)_

Reply:
I admit it wasn't a good idea to use type assertions here rather than inside the component. Both `BrowserRegionEditor` and `BrowserRegionField` are rendered only if `chrLocation` is not `null`, so I thought it was safe to assert `props.chrLocation`. This would not require to check if `props.chrLocation` is `null` or not in these components.